src/generator/AutoRest.AzureResourceSchema.Tests/Expected/ApiManagement/2016-07-07/Microsoft.ApiManagement/service_apis_operations.md

```diff
@@ -0,0 +1,141 @@
+# Microsoft.ApiManagement/service/apis/operations template reference
+API Version: 2016-07-07
+## Template format
+
+To create a Microsoft.ApiManagement/service/apis/operations resource, add the following JSON to the resources section of your template.
+
+```json
+{
+  "type": "Microsoft.ApiManagement/service/apis/operations",
+  "apiVersion": "2016-07-07",
+  "OperationContract": {
+    "id": "string",
+    "name": "string",
+    "method": "string",
+    "urlTemplate": "string",
+    "templateParameters": [
+      {
+        "name": "string",
+        "description": "string",
+        "type": "string",
+        "defaultValue": "string",
+        "required": boolean,
```

Comment:
Shouldn't this be surrounded by quotes?
edit: Along with a bunch of other examples.